### ISSUE TITLE
Keyboard users are forced to use esc or tab through the calendar to use the datepicker and then move focus to next element..

### ISSUE THREAD

**Issue by jason-capsule42** [AlaskaAirlines/auro-datepicker#102]
## Describe the bug

A keyboard only user should be able to tab to the datepicker, type a date, hit tab once and go to the next focusable element on the page.

Because we auto-open the bib when you start typing, a keyboard user is forced to hit the esc key or tab all the way through the calendar.

## To Reproduce

1. Go to http://auro.alaskaair.com/components/auro/datepicker
1. Click use the tab key to get to a datepicker
1. Type a date
1. hit tab again

## Expected behavior

I would like to be able to just type a date and then move one without having to manually escape the calendar.

**Comment by Patrick-Daly-AA** [comment]
@leeejune to look over this issue with Kayla.

**Comment by Patrick-Daly-AA** [comment]
Will review after completion of https://github.com/AlaskaAirlines/auro-dropdown/issues/183 And/or native popover support implementation.
